vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -126,17 +126,19 @@ run_stress_test(
 
 ### Output files
 The output files are written into a subdirectory of the path indicated as
-`output_path` in the function call. The name of the subdirectory is the
-timestamp of the analysis. It will be generated automatically.
+`output_path` in the function call. The name of the subdirectory is created from
+the timestamp of the analysis and a suffix. The suffix is `_standard` if
+stresstest is run with a fixed set of parameters. If a sensitivity analysis (see
```

Comment:
while I may have used a similar formulation somewhere, I feel like "fixed set of parameters" is not intuitively clear to mean: one parameter per argument. "The suffix `_standard` is used when each argument uses a single parameter value."?

Reply:
done